plugins/content/caption/base/lib/api/KalturaCaptionAsset.php

```diff
@@ -6,6 +6,9 @@
  */
 class KalturaCaptionAsset extends KalturaAsset
 {
```

Comment:
This entire logic must be encapsulated into ZoomPlugin, when adding new transcription file